anchor/message_sender/src/network.rs

```diff
@@ -39,6 +53,14 @@ impl<S: SlotClock + 'static, D: DutiesProvider> MessageSender for Arc<NetworkMes
         committee_id: CommitteeId,
         additional_message_callback: Option<Box<MessageCallback>>,
     ) -> Result<(), Error> {
+        // Check if in doppelgänger monitoring period - silently drop
+        if let Some(dg) = &self.doppelganger_service
+            && dg.is_monitoring()
```

Comment:
> https://github.com/sigp/anchor/pull/692#issuecomment-3436215337

@dknopik reallly good point. Made me realize this line is wrong, we shouldn't send any message for the whole dg process (including the grace period). Changing this line should avoid the issue you mentioned, right?